h/api/schemas.py

```diff
@@ -90,9 +205,75 @@ class CreateAnnotationSchema(object):
 
     def __init__(self, request):
         self.request = request
-        self.structure = AnnotationSchema()
+        if self.request.feature('postgres_write'):
+            self.structure = AnnotationSchema()
+        else:
+            self.structure = LegacyAnnotationSchema()
 
     def validate(self, data):
```

Comment:
Oh damn, I forgot that the feature flag was supposed to mean write to both storage backends together, this code is intended (when combined with the next pull request) to write to postgres only when the feature flag is on.

Hopefully it won't be too difficult to fix it to write different data to each instead though..